### PR TITLE
Create sensor.rtorrent.markdown

### PR DESCRIPTION
**Pull request by riptl**
**Description:**
Description for my new rtorrent sensor

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant):** home-assistant/home-assistant#17421

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.rtorrent.markdown [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+description: "Instructions on how to integrate rtorrent sensors within Home Assistant."
+date: 2018-10-14 05:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Downloading
+ha_release: 0.57
+ha_iot_class: "Local Polling"
+---
```
Comment (frenck) [inline source/_components/sensor.rtorrent.markdown:14]: `logo` is missing.

File: source/_components/sensor.rtorrent.markdown [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+# Example configuration.yaml entry
+sensor:
+  - platform: rtorrent
+    url: 'http://<user>:<password>@<host>:<port>/RPC2'
+    monitored_variables:
+      - 'current_status'
+      - 'download_speed'
+      - 'upload_speed'
+```
+
```
Comment (frenck) [inline source/_components/sensor.rtorrent.markdown:30]: Remove additional empty line.

File: source/_components/sensor.rtorrent.markdown [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Downloading
+ha_release: 0.57
+ha_iot_class: "Local Polling"
+---
+
+
```
Comment (frenck) [inline source/_components/sensor.rtorrent.markdown]: Remove additional empty line.

File: source/_components/sensor.rtorrent.markdown [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+      - 'current_status'
+      - 'download_speed'
+      - 'upload_speed'
+```
+
+
+This sensor requires the rtorrent XMLRPC API exposed on HTTP (by default only SCGI).
+([arch-rtorrentvpn](https://github.com/binhex/arch-rtorrentvpn) provides that API under the URL `http://admin:rutorrent@127.0.0.1:9080/RPC2`)
+
+Configuration variables:
```
Comment (frenck) [inline source/_components/sensor.rtorrent.markdown]: Please use the configuration tags, for more information please see:
https://home-assistant.io/developers/documentation/create_page/#configuration

**Review by frenck — CHANGES_REQUESTED** [review]
Thanks for you PR @terorie!
I've reviewed it and left you some comments. Could you please take a look? 👍

**Comment by riptl** [comment]
@frenck Just added the configuration tags and updated the description a bit.
I also set `ha_release` to `0.80`. I hope this is correct.

Thanks for taking a look 👍

**Review by frenck — DISMISSED** [review]

**Comment by frenck** [comment]
Thanks, @terorie!
0.80 is not correct since it has been shipped already.
Nevertheless, even when updating to 0.81, the question remains if it ships in that version (it might as well be the version after that.).

✅ Approved. Can be merged as soon as the parent PR gets merged.
⚠️ The `ha_release` might need an update before merging this one in.

**Review by tjorim — CHANGES_REQUESTED** [review]
Thanks for your first-time PR here!
Maybe I'm just nitpicking but please put the `description` above `required` in the configuration-section.

**Comment by riptl** [comment]
@tjorim Okay, I did so. Btw, I was using https://github.com/home-assistant/home-assistant.io/blob/next/source/_components/sensor.deluge.markdown as guidance, so maybe it should be updated there too then?

**Comment by frenck** [comment]
@terorie All documentation is currently updated to the new format. Since we have a couple of pages, this takes some time.

**Review by frenck — DISMISSED** [review]

File: source/_components/sensor.rtorrent.markdown [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+title: "rtorrent Sensor"
+description: "Instructions on how to integrate rtorrent sensors within Home Assistant."
+date: 2018-10-14 05:40
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: rtorrent.png
+ha_category: Downloading
+ha_release: 0.80
```
Comment (MartinHjelmare) [inline source/_components/sensor.rtorrent.markdown]: 0.81

File: source/_components/sensor.rtorrent.markdown [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+ha_iot_class: "Local Polling"
+---
+
+The `rtorrent` platform allows you to monitor your downloads with [rtorrent](https://rakshasa.github.io/rtorrent/) from within Home Assistant and setup automation based on the information.
```
Comment (MartinHjelmare) [inline source/_components/sensor.rtorrent.markdown]: ... and set up automations based on the information.

File: source/_components/sensor.rtorrent.markdown [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+      - 'current_status'
+      - 'download_speed'
+      - 'upload_speed'
+```
+
+This sensor requires the rtorrent XMLRPC API exposed on an HTTP interface.
+Note that for security reasons, simply using the SCGI interface (default `localhost:5000`) of rtorrent won't work.
+The [official reference](https://github.com/rakshasa/rtorrent/wiki/RPC-Setup-XMLRPC) describes how to set up that HTTP interface.
+
+Alternatively, the [arch-rtorrentvpn](https://github.com/binhex/arch-rtorrentvpn) container can be used with `url ` set to `http://admin:rutorrent@127.0.0.1:9080/RPC2`.
```
Comment (MartinHjelmare) [inline source/_components/sensor.rtorrent.markdown]: `url ` -> `url`

File: source/_components/sensor.rtorrent.markdown [hunk 8]
```diff
@@ -0,0 +1,57 @@
...
+
+{% configuration %}
+url:
+  description: The URL to the HTTP endpoint of the rtorrent XMLRPC API.
+  required: true
+  type: string
+name:
+  description: The name to user when displaying this rtorrent instance.
```
Comment (MartinHjelmare) [inline source/_components/sensor.rtorrent.markdown]: The name to use...

File: source/_components/sensor.rtorrent.markdown [hunk 9]
```diff
@@ -0,0 +1,57 @@
...
+  required: false
+  type: string
+monitored_variables:
+  description: Conditions to display in the frontend.
```
Comment (MartinHjelmare) [inline source/_components/sensor.rtorrent.markdown]: These doesn't have anything to do with the frontend per se. These decide what conditions should be tracked as entity states.

**Review by frenck — APPROVED** [review]

**Pull request merged**